presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -4053,6 +4053,32 @@ public void testCollectColumnStatisticsOnInsert()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCollectColumnStatisticsOnInsertToEmptyTable()
```

Comment:
add a test where the table is empty and it's `ANALYZE`-d, but on one column (of two) only

Reply:
Done.